Document Process Returns as field and auto-release behavior for Shopify refund documents (#1088)
MicrosoftDocs/dynamics365smb-docs: business-central/shopify/synchronize-orders.md

* **Import only** specifies that you import information, but you manually create the corresponding credit memo.
* **Auto create credit memo** specifies that you import information and [!INCLUDE[prod_short](../includes/prod_short.md)] automatically creates the credit memos. This option requires that you turn on the **Auto Create Sales Order** toggle.

> When **Auto Create Sales Document** is selected, the **Process Returns as** field on the **Shopify Shop Card** controls whether the connector creates a sales credit memo or a sales return order. The created document is automatically released after the lines are populated, regardless of the **Auto Release Sales Orders** setting on the shop card.

Reviewer: ~~**Terminology (important):** The toggle is named **Auto Release Sales Order** (singular) per the Shopify Shop Card.~~

**Correction:** I was wrong here — the UI field name is **Auto Release Sales Orders** (plural). The contributor's original text is correct. Please disregard this suggestion. Apologies for the noise!

> > [!NOTE]
> > The connector can only create the sales document if the original Shopify order was already processed into a [!INCLUDE [prod_short](../includes/prod_short.md)] sales document. If the original order hasn't been processed yet, the refund is imported but the sales document creation is skipped and an error is recorded on the refund.

Reviewer: **Accuracy (minor):** The phrase ""an error is recorded on the refund"" is a specific behavioral claim. If this means an entry appears in the Shopify Status Log or on the refund record itself, consider being more precise (e.g., ""the sales document creation is skipped and logged in the Shopify Status Log""). If this is accurate as-is per the codeunit behavior, it's fine to keep.

```suggestion
> [!NOTE]
> The connector can only create the sales document if the original Shopify order was already processed into a [!INCLUDE [prod_short](../includes/prod_short.md)] sales document. If the original order hasn't been processed yet, the refund is imported but the sales document isn't created.
```
